### PR TITLE
Fixed paragraphs in translation #45

### PR DESCRIPTION
**Pull request by Nautman**
I'm not entirely sure this is the wanted fix, but I'll let you decide @Zigolox.

### CODE REVIEW

**Comment by Nautman** [comment]
I have added you as a reviewer @Zigolox. You can [do a suggested change](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/incorporating-feedback-in-your-pull-request) for the text that you would like.

**Comment by Zigolox** [comment]
I have fucked it up. please have mercy on my soul. I just want student's and paragraphs to be correct

**Comment by Nautman** [comment]
No worries at all :)

I have pushed a fix.

File: src/features/recommendation/index.jsx [hunk 1]
```diff
@@ -89,13 +87,18 @@ const Recommendation = () => {
...
                 <p>
                   Thank you for taking interest in writing a letter of
-                  recommendation! The letter must be written, signed and sent by
-                  the students teacher, coach or such. Maximum 1 page per
-                  letter. The student will not be able to see the letter, but
-                  will receive a notification once it is uploaded. The file size
+                  recommendation!
+                </p>
+                <p>
+                  The letter must be written, signed and sent by the student's
```
Comment (Zigolox) [inline src/features/recommendation/index.jsx]:
```suggestion
                  The letter must be written, signed and sent by the students
```

**Review by Zigolox — APPROVED** [review]

**Pull request merged**